bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -215,7 +215,7 @@ public Response getPageData(@Context HttpHeaders headers,
             // Make the REST-API pseudo-compatible with openHAB 1.x
             // The client asks Atmosphere for server push functionality,
             // so we do a simply listening for changes on the appropriate items
-            // The blocking has a timeout of 30 seconds. If this timeout is reached,
+            // The blocking has a timeout of 30 seconds. If this timeout expired,
```

Comment:
I do not actually understand what you mean by "timeout expired". I would have assumed that you mean by this that we have hit the timeout - but then it is the same as "is reached" and I wonder why you have changed it? Imho mixing "expiring" and "timout" is not helpful to explain what has happened...